velox/functions/prestosql/aggregates/MinMaxAggregates.cpp

```diff
@@ -180,6 +180,31 @@ class MinAggregate : public MinMaxAggregate<T> {
     }
   }
 
+  bool supportsToIntermediate() const override {
+    return true;
+  }
+
+  void toIntermediate(
+      const SelectivityVector& rows,
+      std::vector<VectorPtr>& args,
+      VectorPtr& result) const override {
+    const auto& input = args[0];
+    if (rows.isAllSelected()) {
+      result = input;
+      return;
+    }
+
+    auto* pool = BaseAggregate::allocator_->pool();
+
+    result = BaseVector::create(input->type(), rows.size(), pool);
+    result->copy(input.get(), 0, 0, rows.size());
```

Comment:
shouldn't you only copy active rows here?
```
result->copy(input.get(), rows, nullptr);
```

Reply:
I feel it is more efficient to copy the whole vector, then set a few null flags.

Reply:
I remember a discussion at some point that in some cases vector elements nulled out might be non-copyable (like garbage string views). I'm assuming we ensure that these elements are always copyable now?